geode-management/src/main/java/org/apache/geode/management/runtime/RebalanceResult.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.runtime;
+
+import java.util.Map;
+
+import org.apache.geode.annotations.Experimental;
+import org.apache.geode.management.api.JsonSerializable;
+import org.apache.geode.management.operation.RebalanceOperation;
+
+/**
+ * the final results of a {@link RebalanceOperation}
+ */
+@Experimental
+public interface RebalanceResult extends JsonSerializable {
+  /**
+   * @return for each region, the statistics reported by rebalance
+   */
+  Map<String, PerRegionStats> getRebalanceStats();
+
+  interface PerRegionStats extends JsonSerializable {
+    long getBucketCreateBytes();
+
+    long getBucketCreateTimeInMilliseconds();
```

Comment:
This interface and its methods need javadocs since they are part of the external java api. You could cut and paste them from the interface the geode core has.

Reply:
done